README.md

```diff
@@ -1,2 +1,67 @@
-# rails_budget_app
-ruby on rails capstone project.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# BUDGET_APP
+
+## Built With
+
+- Ruby on rails,
+- CSS3
+
+Major language
+- Ruby
+
+
+## Code demo
+[Demo video](https://www.loom.com/share/aaa84d022edf45abbafe53102a44bd0a)
+
+## Live
+[Live](https://my-budget-app-1.herokuapp.com/)
```

Comment:
- [ ] Your live demo is not working. As the previous reviewer mentioned you must have a working live demo to present. 


Requirement:
> The student deploys the application using Heroku, Netlify, or GitHub Pages.

Reply:
Thanks, @Rustamxon7 for the feedback. I am doing my best to have a working live demo. However, the live video demo is working properly. Thanks!

Reply:
Ok, if you are deploying the project to heroku. Make sure you also run `rails db:migrate` as well before opening the project. Also, checking the log will also help... `heroku logs --tail` did you try this?